pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -33,13 +34,22 @@ func getDefaultInstrumentation(agentConfig *adapters.CwaConfig) (*v1alpha1.Instr
 	if !ok {
 		return nil, errors.New("unable to determine python instrumentation image")
 	}
+	dotNetInstrumentationImage, ok := os.LookupEnv("AUTO_INSTRUMENTATION_DOTNET")
+	if !ok {
+		return nil, errors.New("unable to determine dotnet instrumentation image")
+	}
+
+	cloudwatchAgentServiceEndpoint := "cloudwatch-agent.amazon-cloudwatch"
+	if isWindowsPod {
+		cloudwatchAgentServiceEndpoint = "cloudwatch-agent-windows-headless.amazon-cloudwatch.svc.cluster.local"
```

Comment:
we need to use headless endpoint for windows nodes

Reply:
nitpick - worth adding a comment on why does the service endpoint differ for windows